controllers/reviews.controllers.games.js

```diff
@@ -0,0 +1,20 @@
+const {
+  selectReviews,
+  selectReviewsById,
+} = require("../models/reviews.models.games");
+
+exports.getReviews = (req, res) => {
+  selectReviews().then((reviews) => {
+    res.status(200).send(reviews);
+  });
+};
```

Comment:
I can see a lot of code for the getReviews endpoint in this PR, this is an entirely unrelated ticket so the code for that shouldn't be here. Before you merge I'd like you to remove code relating to getReviews from this branch (if you've already gotten it merged in or it's already PR'd on a diff branch then it won't be a problem, if the getReviews endpoint only exists on this branch then make a new branch and PR for that work)